test/c_data_integration/src/comparison.cpp

```diff
@@ -54,27 +54,28 @@ namespace sparrow::c_data_integration
             );
         }
 
-        if ((schema->name != nullptr) || (schema_from_json->name != nullptr))
-        {
-            if ((schema->name != nullptr) != (schema_from_json->name != nullptr))
-            {
-                differences.push_back(
-                    prefix_with_name
-                    + " name mismatch: pointer=" + std::string(schema->name ? schema->name : "nullptr")
-                    + " vs json=" + std::string(schema_from_json->name ? schema_from_json->name : "nullptr")
-                );
-            }
-            else if (schema->name != nullptr && schema_from_json->name != nullptr)
-            {
-                if (std::strcmp(schema->name, schema_from_json->name) != 0)
-                {
-                    differences.push_back(
-                        prefix_with_name + " name mismatch: pointer=" + std::string(schema->name)
-                        + " vs json=" + std::string(schema_from_json->name)
-                    );
-                }
-            }
-        }
+        // if ((schema->name != nullptr) || (schema_from_json->name != nullptr))
+        // {
+        //     if ((schema->name != nullptr) != (schema_from_json->name != nullptr))
+        //     {
+        //         differences.push_back(
+        //             prefix_with_name
+        //             + " name mismatch: pointer=" + std::string(schema->name ? schema->name : "nullptr")
+        //             + " vs json=" + std::string(schema_from_json->name ? schema_from_json->name :
+        //             "nullptr")
+        //         );
+        //     }
+        //     else if (schema->name != nullptr && schema_from_json->name != nullptr)
+        //     {
+        //         if (std::strcmp(schema->name, schema_from_json->name) != 0)
+        //         {
+        //             differences.push_back(
+        //                 prefix_with_name + " name mismatch: pointer=" + std::string(schema->name)
+        //                 + " vs json=" + std::string(schema_from_json->name)
+        //             );
+        //         }
+        //     }
+        // }
```

Comment:
Wy commenting this?

Reply:
I have to remove it, but the thing is that the dictionary name produced by the C++ implementation doesn't follow the other layout and it's a pain to compare